Documentation/Developer.md

```diff
@@ -3,7 +3,7 @@
 
 ## Communication
 
- * General questions and discussion: [portability-discuss@googlegroups.com](mailto:portability-discuss@googlegroups.com)
+ * General questions and discussion: [dtp-discuss@googlegroups.com](mailto:dtp-discuss@googlegroups.com)
  * [Slack channel](https://portability.slack.com), email
    [portability-maintainers@googlegroups.com](mailto:portability-maintainers@googlegroups.com) for invitation
```

Comment:
assuming these will also change?

Reply:
Yes, I'll follow up with the owner of the list to rename and will update accordingly.